cfgov/v1/templates/v1/page_metadata_report.html

```diff
@@ -1,4 +1,13 @@
 {% extends 'wagtailadmin/reports/base_page_report.html' %}
+{% load i18n wagtailadmin_tags %}
+
+{% block actions %}
+    {% if view.list_export %}
+        <div class="button match-width">
```

Comment:
Looks like the `button` class comes with some extra padding that's not needed if there's no dropdown. I think changing that to `actionbutton` should get it looking right (that's the class that's used for buttons like "add an image" in the image manager).

```suggestion
        <div class="actionbutton">
```